USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -0,0 +1,250 @@
+        {
+            
+            switch (input)
+            {
+                case PMarker _:
+                    
+                        XWPFParagraph newParagraph = newDoc.CreateParagraph();
+                        foreach (Marker marker in input.Contents)
+                        {
+                            RenderMarker(marker, newParagraph);
+                        }
+                    break;
+                case CMarker cMarker:
+                    XWPFParagraph newChapter = newDoc.CreateParagraph();
+                    XWPFRun chapterMarker = newChapter.CreateRun();
+                    chapterMarker.SetText($"{cMarker.Number}");
+
+                    chapterMarker.FontSize = 24;
+
+                    XWPFParagraph chapterVerses = newDoc.CreateParagraph();
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker, chapterVerses);
+                    }
+                    
+
+                    RenderFootnotes();
+
+                    // Page breaks after each chapter
+                    if (separateChapters)
+                    {
+                        newDoc.CreateParagraph().CreateRun().AddBreak(BreakType.PAGE);
+                    }
+
+                    break;
+                case VMarker vMarker:
+                    
+
+                    XWPFRun verseMarker = parentParagraph.CreateRun();
+                    verseMarker.SetText($"  {vMarker.VerseCharacter}  ");
+                    verseMarker.Subscript = VerticalAlign.SUPERSCRIPT;
+
+
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker, parentParagraph);
+                    }
+                    break;
+                case QMarker qMarker:
+                    XWPFParagraph poetryParagraph = newDoc.CreateParagraph();
+
+                    // Not sure if indentation works
+                    poetryParagraph.IndentationLeft =qMarker.Depth;
+
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker,poetryParagraph);
+                    }
+
+                    break;
+                case MMarker mMarker:
+                    break;
+                case TextBlock textBlock:
+                    XWPFRun blockText = parentParagraph.CreateRun();
+                    blockText.SetText(textBlock.Text);
+                    blockText.FontSize = 16;
+
+                    if (isBold)
+                    {
+                        blockText.IsBold = true;
+                    }
+                    if (isItalics)
+                    {
+                        blockText.IsItalic = true;
+                    }
+
+                    break;
+                case BDMarker bdMarker:
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker,parentParagraph,true);
```

Comment:
It would make more sense if you were to use `isBold: true` since it is more clear as to what is going on.

Reply:
Yup! I didn't know you could do that. 😄 That is very useful